.scss-lint.yml

```diff
@@ -90,10 +90,11 @@ linters:
     max_depth: 2
     severity: warning
   SelectorFormat:
-    enabled: true
-    convention: hyphenated_lowercase
+    enabled: false # strict_BEM doesn't seem to be supported by Hound
+    convention: strict_BEM
```

Comment:
Does this work for you locally? Hound should just be calling `scss-lint`. If there's a bug, we should report it.